### PR TITLE
Release v0.0.7

### PR DESCRIPTION
**Pull request by ajfloeder**

### CODE REVIEW

**Review by bdevcich — CHANGES_REQUESTED** [review]
This needs to target the release branch, not master.

File: config/repositories.yaml [hunk 1]
```diff
@@ -33,14 +33,14 @@ repositories:
     master: https://ghcr.io/hewlettpackard/lustre-csi-driver
     useRemoteK: true
     remoteReference:
-      build: v0.0.6
+      build: 0.0.7
```
Comment (bdevcich) [inline config/repositories.yaml:36]: I think these need the `v`. @roehrich-hpe?

Comment (ajfloeder) [reply config/repositories.yaml:36]: Good catch.

**Review by bdevcich — APPROVED** [review]

**Comment by roehrich-hpe** [comment]
I cloned a fresh copy of the repo, checked out this PR, and did the ./update.sh.  This is what I see for the submodules:
```
$ git submodule status    
+4b67213ec79a885df815e9ba5fac6a0379db839b dws (v0.0.0-283-g4b67213e)
+e58b06cb55bc91e756ccbb4d9e60abbb8a40a8b6 lustre-csi-driver (v0.0.0-79-ge58b06c)
+eb84599f9630780c902190a281a70564e5221638 lustre-fs-operator (v0.0.0-83-geb84599)
+67f89bd426d1f47799043fe4eea27fca68b16b89 nnf-dm (v0.0.0-198-g67f89bd)
+e25619962e0d5b9e7ff0d4f36178ddc3dbcc0ab1 nnf-integration-test (v0.0.0-61-ge256199)
+871b2caa8c9c45e206fc3728ecc0e993d7a2e362 nnf-sos (v0.0.0-456-g871b2caa)
```

I was expecting that each one would include the commit that has the annotated tag.

**Comment by bdevcich** [comment]
> I was expecting that each one would include the commit that has the annotated tag.

When I use LLNL's process to checkout this branch, I see the submodules at the right tags:

```shell
export NNF_VERSION=v0.0.7
 
git clone --recurse-submodules git@github.com:NearNodeFlash/nnf-deploy nnf-deploy-$NNF_VERSION
cd nnf-deploy-$NNF_VERSION
```

Since the tag doesn't exist yet, I checkout the `release-v0.0.7` branch and update the submodules:

```shell
git checkout release-v0.0.7
git submodule update
```

Then I get all the tags:

```shell
$ git submodule status
 071e4187ad6cfebc2ea582be3bec47d2124b75ad dws (v0.0.15)
 daa1cff12120f0aa991015c95b7527ace7f33ea2 lustre-csi-driver (v0.0.7)
 f983003e9a022ea1db56f9d03b89710f2c8110fd lustre-fs-operator (v0.0.7)
 a56e780213dbb73e141f4b2c5676fbea66a40f92 nnf-dm (v0.0.7)
 a3be2b1b63474d85364e0f8493bb43e9c5d29188 nnf-integration-test (v0.0.4)
 226cbe05c4a5e5180ba4d20fc48158ed13291ec0 nnf-sos (v0.1.1)
```

**Comment by roehrich-hpe** [comment]
> > I was expecting that each one would include the commit that has the annotated tag.
> 
> When I use LLNL's process to checkout this branch, I see the submodules at the right tags:

Yep, that finds the right stuff.

Gotta be careful with that ./update.sh tool.

**Review by roehrich-hpe — APPROVED** [review]

**Pull request merged**